airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_41_01_001__AddActorDefinitionConfigInjection.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import static org.jooq.impl.DSL.currentOffsetDateTime;
+import static org.jooq.impl.DSL.foreignKey;
+import static org.jooq.impl.DSL.primaryKey;
+import static org.jooq.impl.DSL.unique;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.JSONB;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_41_01_001__AddActorDefinitionConfigInjection extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_41_01_001__AddActorDefinitionConfigInjection.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    // Warning: please do not use any jOOQ generated code to write a migration.
+    // As database schema changes, the generated jOOQ code can be deprecated. So
+    // old migration may not compile if there is any generated code.
+    try (final DSLContext ctx = DSL.using(context.getConnection())) {
```

Comment:
Heads up that this is going to conflict with an open PR that I have:  https://github.com/airbytehq/airbyte-platform/pull/128/.  We are attempting to update to a modern version of jOOQ (we are two years behind in terms of releases).  One of the changes is that the `DSLContext` class is no longer auto-closeable, so if my PR lands first, you will simply need to remove the try-with-resource block.